elbow_angle.py

```diff
@@ -169,8 +177,28 @@ def elbow_angle(obj, light='L', heavy='H', limit_l=107, limit_h=113, draw=0):
     # the C-alpha atoms of limit_l and limit_h of the original fab
     hinge_l_sel = "%s//%s/%s/CA" % (obj, light, limit_l)
     hinge_h_sel = "%s//%s/%s/CA" % (obj, heavy, limit_h)
-    hinge_l = cmd.get_atom_coords(hinge_l_sel)
-    hinge_h = cmd.get_atom_coords(hinge_h_sel)
+
+    def safe_get_atom_coords(sel):
+        '''Get atom coords or return None if not found.
+
+        Works around a bug in cmd.get_atom_coords() where it fails silently if
+        the atom in the selection does not exist.
+
+        See: https://github.com/schrodinger/pymol-open-source/issues/74
+        '''
+        try:
+            cmd.id_atom(sel)
```

Comment:
is calling `cmd.id_atom` necessary? What happens without it? `cmd. get_atom_coords` _should_ raise `CmdException` if not exactly one atom is selection.

Reply:
You're right. It works just the same without `id_atom()`.  I just didn't realize anything was being raised because the CmdException is apparently caught and nothing happens?  Not sure exactly how this is set up, but I think what you suggested in the other issue (pymol-open-source#74) would certainly help.

Reply:
What is the recommended way to see what happens to that error?  If I put `raise` instead of `return None` inside the `safe_get_atom_coords` function, there is no indication of an error, either, everything just stops silently without a traceback.

Reply:
I recommend this:
```python
    try:
        hinge_l = cmd.get_atom_coords(hinge_l_sel)
        hinge_h = cmd.get_atom_coords(hinge_h_sel)
    except pymol.CmdException:
        raise pymol.CmdException('Unable to calculate elbow angle. Please check '
                'your limit and chain selections and try again.')
```
PyMOL will print the `CmdException` error message without traceback (since PyMOL 1.6, before that `CmdException` was always silent).

Reply:
Thanks, Thomas, that looks great.